app/mailers/hypertension_fast_track_pilot_mailer.rb

```diff
@@ -5,22 +5,14 @@ def build(submission)
     @submission = submission
     @disability = submission.form.dig('form526', 'form526', 'disabilities')&.first
     @disability_code = @disability['diagnosticCode'] if @disability
-    @rrd_claim_processed = RapidReadyForDecision::Form526BaseJob.rrd_claim_processed?(submission)
-    @bp_readings_count = submission.form.dig('rrd_med_stats', 'bp_readings_count') || 'N/A'
-
-    subject =
-      if @rrd_claim_processed
-        'RRD claim - Processed'
-      else
-        'RRD claim - Not processed'
-      end
-    subject += " - #{@disability_code}"
+    @rrd_status = RapidReadyForDecision::Form526BaseJob.rrd_status(submission)
+    @bp_readings_count = submission.form.dig('rrd_metadata', 'med_stats', 'bp_readings_count') || 'N/A'
 
     template = File.read('app/mailers/views/hypertension_fast_track_pilot_mailer.erb.html')
 
     mail(
       to: Settings.rrd.event_tracking.recipients,
-      subject: subject,
+      subject: "RRD claim - #{@rrd_status.to_s.humanize} - #{@disability_code}",
```

Comment:
In line with my original list of things to not do in this PR, I'm relying on `String#humanize` here to easily boil down the status into a single symbol. This does mean `:pending_ep` will display as "Pending ep" 🤷